act/discovery/get_airnow.py

```diff
@@ -0,0 +1,241 @@
+import pandas as pd
+import numpy as np
+import xarray as xr
+
+
+def get_AirNow_forecast(token, date, zipcode=None, latlon=None, distance=25):
+    """
+    This tool will get current or historical AQI values and categories for a
+    reporting area by either Zip code or Lat/Lon coordinate.
+
+    Parameters
+    ----------
+    token : str
+        The access token for accesing the AirNowAPI web server
```

Comment:
One small suggestion - can you add the link here to the AirNowAPI? Just thinking about the user trying to find where to get their login.